### PR TITLE
General code cleanup and improvement

### PR DESCRIPTION
**Pull request by MineKing9534**
## Checklist

 - [x] Read [Contribution Guidelines](https://github.com/SlimeCloud/java-SlimeBot#contributing)
 - [x] Tested the Code
 - [x] This PR is ready to review and merge

## Description:
The Main class was quite unorganized. I tried to clean it up a little by extracting the command registration into its own method. I also fixed the command registration that deleted all commands after creating them before. The input that asks for a token name to use at startup was replaced by the first command line parameter.
The command handling still doesn't work because the listeners in the command classes use #getName instead of #getFullCommandName.
Currently there is no uniform code style. I will format all code with the default IntelliJ style. If @MatitoDA prefers a different one, you can comment it below so I will change it to the style you prefer. 

- [x] Improve startup
- [x] Cleanup command registration
- [x] Fix command handing (waiting for response whether to use a system similar to the current one or a CommandManager)
- [x] Format all code according to a uniform pattern

## Related Issue
N/A

## Other Information
N/A

### CODE REVIEW

**Comment by MatitoDA** [comment]
> Currently there is no uniform code style. I will format all code with the style that is most used. If @MatitoDA prefers a different one, you can comment it below so I will change it to the style you prefer.

can you explain in more detail which style you would use?

**Comment by Byteingpython** [comment]
The Jetbrains default

**Comment by MineKing9534** [comment]
I would use the IntelliJ default that is applied when letting it do a code reformat. I personally also like it more to use 'var' everywhere where it is possible but we can just leave it as it is.

File: src/main/java/com/slimebot/report/assets/Status.java [hunk 1]
```diff
@@ -1,6 +1,12 @@
 package com.slimebot.report.assets;
 
 public enum Status {
-    CLOSED,
-    OPEN
+	CLOSED("Geschlossed"),
```
Comment (MatitoDA) [inline src/main/java/com/slimebot/report/assets/Status.java:4]: typo

File: src/main/java/com/slimebot/report/assets/Type.java [hunk 2]
```diff
@@ -1,7 +1,13 @@
 package com.slimebot.report.assets;
 
 public enum Type {
-    MSG,
-    USER
+	MSG("Nachricht"),
+	USER("USer");
```
Comment (MatitoDA) [inline src/main/java/com/slimebot/report/assets/Type.java:5]: typo

**Review by MatitoDA — CHANGES_REQUESTED** [review]

**Comment by Deliasama** [comment]
Du hast das Emum verkrüppelt die funktionen sind für noch folgende projekte mit emoji

**Comment by MineKing9534** [comment]
Your requested changes were made and I resolved the merge conflicts against master. This is ready for a new review.

**Comment by MatitoDA** [comment]
Whenever I run your code longer than 2 min, this error comes up

```
14:26:36 main                       Main                 INFO   Bot Version: 2.0
14:26:36 main                       Main                 INFO   test-Bot wird gestartet...
14:26:36 main                       JDA                  INFO   Login Successful!
14:26:37 JDA MainWS-ReadThread      WebSocketClient      INFO   Connected to WebSocket
14:26:37 JDA MainWS-ReadThread      JDA                  INFO   Finished Loading!
14:26:37 JDA MainWS-ReadThread      SpotifyListener      INFO   überprüfe auf neue Releases
14:28:40 JDA Gateway-Worker 1       WebSocketClient      WARN   Missed 2 heartbeats! Trying to reconnect...
java.lang.RuntimeException: java.lang.InterruptedException
	at com.slimebot.alerts.spotify.PodcastListener.<init>(PodcastListener.java:33)
	at com.slimebot.alerts.spotify.SpotifyListenerManager.register(SpotifyListenerManager.java:53)
	at com.slimebot.events.ReadyEvent.onReady(ReadyEvent.java:20)
	at net.dv8tion.jda.api.hooks.ListenerAdapter.onEvent(ListenerAdapter.java:442)
	at net.dv8tion.jda.api.hooks.InterfacedEventManager.handle(InterfacedEventManager.java:96)
	at net.dv8tion.jda.internal.hooks.EventManagerProxy.handleInternally(EventManagerProxy.java:88)
	at net.dv8tion.jda.internal.hooks.EventManagerProxy.handle(EventManagerProxy.java:70)
	at net.dv8tion.jda.internal.JDAImpl.handleEvent(JDAImpl.java:176)
	at net.dv8tion.jda.internal.requests.WebSocketClient.ready(WebSocketClient.java:212)
	at net.dv8tion.jda.internal.handle.GuildSetupController.checkReady(GuildSetupController.java:115)
	at net.dv8tion.jda.internal.handle.GuildSetupController.ready(GuildSetupController.java:102)
	at net.dv8tion.jda.internal.handle.GuildSetupNode.completeSetup(GuildSetupNode.java:377)
	at net.dv8tion.jda.internal.handle.GuildSetupNode.handleMemberChunk(GuildSetupNode.java:260)
	at net.dv8tion.jda.internal.handle.GuildSetupNode.ensureMembers(GuildSetupNode.java:394)
	at net.dv8tion.jda.internal.handle.GuildSetupNode.handleCreate(GuildSetupNode.java:220)
	at net.dv8tion.jda.internal.handle.GuildSetupController.onCreate(GuildSetupController.java:170)
	at net.dv8tion.jda.internal.handle.GuildCreateHandler.handleInternally(GuildCreateHandler.java:44)
	at net.dv8tion.jda.internal.handle.SocketHandler.handle(SocketHandler.java:39)
	at net.dv8tion.jda.internal.requests.WebSocketClient.onDispatch(WebSocketClient.java:1014)
	at net.dv8tion.jda.internal.requests.WebSocketClient.onEvent(WebSocketClient.java:900)
	at net.dv8tion.jda.internal.requests.WebSocketClient.handleEvent(WebSocketClient.java:878)
	at net.dv8tion.jda.internal.requests.WebSocketClient.onBinaryMessage(WebSocketClient.java:1053)
	at com.neovisionaries.ws.client.ListenerManager.callOnBinaryMessage(ListenerManager.java:385)
	at com.neovisionaries.ws.client.ReadingThread.callOnBinaryMessage(ReadingThread.java:276)
	at com.neovisionaries.ws.client.ReadingThread.handleBinaryFrame(ReadingThread.java:996)
	at com.neovisionaries.ws.client.ReadingThread.handleFrame(ReadingThread.java:755)
	at com.neovisionaries.ws.client.ReadingThread.main(ReadingThread.java:108)
	at com.neovisionaries.ws.client.ReadingThread.runMain(ReadingThread.java:64)
	at com.neovisionaries.ws.client.WebSocketThread.run(WebSocketThread.java:45)
Caused by: java.lang.InterruptedException
	at java.base/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1638)
	at net.dv8tion.jda.internal.JDAImpl.awaitStatus(JDAImpl.java:526)
	at net.dv8tion.jda.api.JDA.awaitStatus(JDA.java:328)
	at net.dv8tion.jda.api.JDA.awaitReady(JDA.java:378)
	at com.slimebot.alerts.spotify.PodcastListener.<init>(PodcastListener.java:30)
	... 28 more
```

**Comment by MineKing9534** [comment]
This doesn't happen for me, I will look into it in a moment.

**Comment by MineKing9534** [comment]
Okay, I found a JDA#awaitReady that shouldn't be there, does it work now?

**Review by MatitoDA — APPROVED** [review]

**Pull request merged**